Dockerfile

```diff
@@ -6,24 +6,11 @@ VOLUME /config
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-RUN pip3 install --no-cache-dir colorlog cython
-
-# For the nmap tracker, bluetooth tracker, Z-Wave, tellstick
-RUN echo "deb http://download.telldus.com/debian/ stable main" >> /etc/apt/sources.list.d/telldus.list && \
-    wget -qO - http://download.telldus.se/debian/telldus-public.key | apt-key add - && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends nmap net-tools cython3 libudev-dev sudo libglib2.0-dev bluetooth libbluetooth-dev \
-            libtelldus-core2 cmake libxrandr-dev swig && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-COPY script/build_python_openzwave script/build_python_openzwave
-RUN script/build_python_openzwave && \
-  mkdir -p /usr/local/share/python-openzwave && \
-  ln -sf /usr/src/app/build/python-openzwave/openzwave/config /usr/local/share/python-openzwave/config
-
-COPY script/build_libcec script/build_libcec
-RUN script/build_libcec
+# Copy build scripts
+COPY script/setup_docker_prereqs script/build_python_openzwave script/build_libcec script/
```

Comment:
Do we want to just copy the whole script directory for simplicity? It's all going to get copied over when the source directory is copied anyway. We could even promote the `COPY . .` line to before running the scripts so everything is available.

Again, docker isn't my forte so I'm not sure if increasing the size of intermediate images is a concern.